spec/lib/notify/notification_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require 'notify/notification'
+
+RSpec.describe 'Notification', type: :mailer do
```

Comment:
do we actually treat Notify as a mailer within the the ActionMailer context? The gem is used as a simple client, not a mailer from the first glance...

Reply:
you are right, it isn't a mailer as such. Have amended.